internal server error when trying to access Task logs

I am getting the following error when trying to access the Task logs:
`{"errors":[{"code":"UNKNOWN","message":"internal server error"}]}`

In the core Pod I see:
```shell
[ERROR] [/server/error/error.go:53]: {"errors":[{"code":"INTERNAL_SERVER_ERROR","message":"failed to get log of task 139: http error: code 500, message {\"code\":10009,\"message\":\"failed to get the job log\",\"details\":\"no log data getter is configured\"}"}]} 
/harbor/src/server/error/error.go:81, github.com/goharbor/harbor/src/server/error.apiError
/harbor/src/server/error/error.go:50, github.com/goharbor/harbor/src/server/error.SendError
/harbor/src/common/api/base.go:67, github.com/goharbor/harbor/src/common/api.(*BaseAPI).RenderError
/harbor/src/common/api/base.go:228, github.com/goharbor/harbor/src/common/api.(*BaseAPI).SendInternalServerError
/harbor/src/core/api/replication_execution.go:238, github.com/goharbor/harbor/src/core/api.(*ReplicationOperationAPI).GetTaskLog
/usr/local/go/src/reflect/value.go:460, reflect.Value.call
/usr/local/go/src/reflect/value.go:321, reflect.Value.Call
/harbor/src/vendor/github.com/astaxie/beego/router.go:853, github.com/astaxie/beego.(*ControllerRegister).ServeHTTP
/harbor/src/server/middleware/transaction/transaction.go:62, github.com/goharbor/harbor/src/server/middleware/transaction.Middleware.func1.1
/harbor/src/lib/orm/orm.go:63, github.com/goharbor/harbor/src/lib/orm.WithTransaction.func1
/harbor/src/server/middleware/transaction/transaction.go:71, github.com/goharbor/harbor/src/server/middleware/transaction.Middleware.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/notification/notification.go:31, github.com/goharbor/harbor/src/server/middleware/notification.Middleware.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/orm/orm.go:53, github.com/goharbor/harbor/src/server/middleware/orm.MiddlewareWithConfig.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/middleware.go:52, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/security/security.go:62, github.com/goharbor/harbor/src/server/middleware/security.Middleware.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/csrf/csrf.go:55, github.com/goharbor/harbor/src/server/middleware/csrf.attach.func1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/vendor/github.com/gorilla/csrf/csrf.go:297, github.com/gorilla/csrf.(*csrf).ServeHTTP
/harbor/src/server/middleware/csrf/csrf.go:75, github.com/goharbor/harbor/src/server/middleware/csrf.Middleware.func2
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/session/session.go:34, github.com/goharbor/harbor/src/server/middleware/session.Middleware.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/log/log.go:33, github.com/goharbor/harbor/src/server/middleware/log.Middleware.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/requestid/requestid.go:37, github.com/goharbor/harbor/src/server/middleware/requestid.Middleware.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/usr/local/go/src/net/http/server.go:2802, net/http.serverHandler.ServeHTTP
/usr/local/go/src/net/http/server.go:1890, net/http.(*conn).serve
```

And in the Job service Pod:
`[ERROR] [/jobservice/api/handler.go:288]: Serve http request 'GET /api/v1/jobs/ffb5444c9ca3353f56f8249d/log' error: 500 {"code":10009,"message":"failed to get the job log","details":"no log data getter is configured"}`

I suspect this may be due to setting `jobservice.jobLogger: stdout`. I prefer this go to STDOUT so that Loki may capture the logs. Is it possible to define more than one logger (ie. `jobservice.jobLogger: "stdout,file"`)? This would solve both use-cases.

--- a/templates/jobservice/jobservice-cm.yaml
+++ b/templates/jobservice/jobservice-cm.yaml
@@ -22,7 +22,7 @@ data:
         namespace: "harbor_job_service_namespace"
         idle_timeout_second: 3600
     job_loggers:
-      {{- if eq .Values.jobservice.jobLogger "file" }}
+      {{- if has "file" .Values.jobservice.jobLoggers }}
       - name: "FILE"
         level: {{ .Values.logLevel | upper }}
         settings: # Customized settings of logger
@@ -31,12 +31,14 @@ data:
           duration: 14 #days
           settings: # Customized settings of sweeper
             work_dir: "/var/log/jobs"
-      {{- else if eq .Values.jobservice.jobLogger "database" }}
+      {{- end }}
+      {{- if has "database" .Values.jobservice.jobLoggers }}
       - name: "DB"
         level: {{ .Values.logLevel | upper }}
         sweeper:
           duration: 14 #days
-      {{- else }}
+      {{- end }}
+      {{- if has "stdout" .Values.jobservice.jobLoggers }}
       - name: "STD_OUTPUT"
         level: {{ .Values.logLevel | upper }}
       {{- end }}

--- a/templates/jobservice/jobservice-dpl.yaml
+++ b/templates/jobservice/jobservice-dpl.yaml
@@ -110,7 +110,7 @@ spec:
         configMap:
           name: "{{ template "harbor.jobservice" . }}"
       - name: job-logs
-        {{- if and .Values.persistence.enabled (eq .Values.jobservice.jobLogger "file") }}
+        {{- if and .Values.persistence.enabled (has "file" .Values.jobservice.jobLogger) }}
         persistentVolumeClaim:
           claimName: {{ .Values.persistence.persistentVolumeClaim.jobservice.existingClaim | default (include "harbor.jobservice" .) }}
         {{- else }}

--- a/templates/jobservice/jobservice-pvc.yaml
+++ b/templates/jobservice/jobservice-pvc.yaml
@@ -1,6 +1,6 @@
 {{- $jobservice := .Values.persistence.persistentVolumeClaim.jobservice -}}
 {{- if and .Values.persistence.enabled (not $jobservice.existingClaim) }}
-{{- if eq .Values.jobservice.jobLogger "file" }}
+{{- if has "file" .Values.jobservice.jobLoggers }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:

--- a/values.yaml
+++ b/values.yaml
@@ -444,7 +444,11 @@ jobservice:
   serviceAccountName: ""
   maxJobWorkers: 10
   # The logger for jobs: "file", "database" or "stdout"
-  jobLogger: file
+  jobLoggers:
+    - file
+    # - database
+    # - stdout
+
   # resources:
   #   requests:
   #     memory: 256Mi

